safe/gis/expressions.py

```diff
@@ -85,15 +85,17 @@ def inasafe_place_value_coefficient(number, feature, parent):
     It needs to be used with inasafe_number_denomination_unit.
     """
     _ = feature, parent  # NOQA
-    if number is None:
+
+    if number >= 0:
+        rounded_number = round_affected_number(
+            number,
+            enable_rounding=True,
+            use_population_rounding=True
+        )
+        value, unit = denomination(rounded_number, 1000)
+        return str(round(value, 1))
+    else:
         return no_data_replacement['string_format']
```

Comment:
Now I'm thinking that we shouldn't provide extra behaviors like this and only do what the function is supposed to do : return the coefficient or NULL. Especially not mixing integers/float AND chars like dash.
If the user wants a dash if the string is empty, then it is a `IF` statement in the QGIS expression. We might have users who doesn't want a dash. It something that can easily add in a QGIS expression. I would like to stay as more generic as possible in these expressions.